src/box/iproto.cc

```diff
@@ -3221,11 +3224,10 @@ net_send_msg(struct cmsg *m)
 	}
 	con->wend = msg->wpos;
 
-	if (con->state == IPROTO_CONNECTION_ALIVE) {
+	if (con->state == IPROTO_CONNECTION_ALIVE)
 		iproto_connection_feed_output(con);
-	} else if (iproto_connection_is_idle(con)) {
+	else if (iproto_connection_is_idle(con))
 		iproto_connection_close(con);
-	}
 	iproto_msg_delete(msg);
```

Comment:
Please avoid unrelated changes. They make the patch difficult to review.

Reply:
Fixed.